.travis.yml

```diff
@@ -15,5 +15,5 @@ cache:
 install:
   - make dependencies
 script:
-  - test "$LINT" -eq 1 && make lint || echo "Skipping lint"
+  - test "$LINT" = 1 && make lint
```

Comment:
I guess we'll find out, but won't this fail the build whenever linting is disabled?

Reply:
Derp. You're right.

Reply:
I think we want something more like

```
test -z "$LINT" || make lint
```